data/authorities.json

```diff
@@ -204,6 +204,46 @@
       }
     }
   },
+  "NV": {
+    "state": {},
+    "utility": {
+      "nv-nv-energy": {
+        "name": "NV Energy"
+      },
+      "nv-wells-rural-electric": {
+        "name": "Wells Rural Electric"
+      },
+      "nv-lincoln-county-power-district-no-1": {
+        "name": "Lincoln County Power District No. 1"
+      },
+      "nv-mt-wheeler-power": {
+        "name": "Mt. Wheeler Power"
+      },
+      "nv-city-of-boulder-city": {
+        "name": "City of Boulder City"
+      },
+      "nv-raft-rural-electric-cooperative": {
+        "name": "Raft Rural Electric Cooperative"
+      },
+      "nv-plumas-sierra-rural-electric-cooperative": {
+        "name": "Plumas-Sierra Rural Electric Cooperative"
+      }
+    },
+    "other": {
+      "nv-csa-reno": {
+        "name": "CSA Reno"
+      },
+      "nv-rural-nevada-development-corporation": {
+        "name": "Rural Nevada Development Corporation"
+      },
+      "nv-nevada-rural-housing": {
+        "name": "Nevada Rural Housing"
+      },
+      "nv-help-of-southern-nevada": {
+        "name": "HELP of Southern Nevada"
+      }
+    }
```

Comment:
these were marked as "other" in the spreadsheet because they're independent nonprofits. is this the right way to represent nonprofits?

Reply:
Hmm, these are tricky.

1. Even if we don't change much else, I don't think we should have an "other" category for AuthorityType. If we do need a separate category, I think we should add some kind of private/nonprofit option.

2. Most of these are eligibility-bounded by county (can't tell with the HELP one). This does seem like an example of authority and eligibility being largely unrelated. They're not County authorities, but they have County eligibility boundaries.

Going to defer to @oyamauchi on how to proceed.

Reply:
This is running up against the exact problem I'm trying to work on now. Here's the story from the beginning:

- Originally, I thought that the "authority" (i.e. the entity offering/administering the incentive) would be coterminous with the set of people who were eligible for the incentive based on location. I.e. state-level authorities would offer incentives to everyone in the state, utilities would offer incentives to their customers, city-level authorities would offer to their residents, etc. Notably, "state-level authority" didn't necessarily mean "part of the state government", it meant any authority who offered incentives to all state residents.
- This has turned out not to be the case. One problem we have in VT and CO is the existence of consortiums of utilities, which have a distinct identity and offer incentives to customers of any of several utilities.
- These nonprofits are another assumption-breaking case: their incentives are actually geographically restricted, by county. (E.g. [this page](https://rndcnv.org/weatherization-services-rural-nevada-development-corporation/) has a list of counties where the incentive is available.)
- So I'm thinking it's time to formally separate these concepts. I'm working on a change to do so for utilities; I think it'll be a new field on incentives that contains a list of utilities.
- Following on from _that_, I'm thinking the distinction between authority types in this file might become moot? The authority type/identity would stop serving as an eligibility criterion at all, and would instead just be a label for who administers the incentive.

So that's my thinking so far; sorry for the wall of text. (Your opinions on this stuff are welcome!) For this PR, I think what you have here is fine, with the understanding that it'll probably change soon.

Reply:
This is really helpful, thank you! One other consideration is that for at least some of these programs (like [NV-76](https://nvrural.org/programs/weatherization/)) the income qualifications also differ by county, in some counties it's 150% of the federal poverty line and in others it's 200%.

After looking at these 4 websites more closely though I actually don't see anywhere where it says what the value of the rebate is, the values in the spreadsheet seem to be out of date. Based on that I think I'm going to remove these anyway

Reply:
Just for visibility, a similar issue presented itself with DC, where a private entity was given the sole authority to offer energy rebates on behalf of the city. DC government confirmed that it would best be labeled "city". (One somewhat related note is that DC recently changed the law so that utilities can also offer rebates to their own customers.)